hera_cal/abscal.py

```diff
@@ -2779,126 +2833,276 @@ def get_d2m_time_map(data_times, data_lsts, model_times, model_lsts, unwrap=True
     return d2m_time_map
 
 
-def abscal_step(gains_to_update, AC, AC_func, AC_kwargs, gain_funcs, gain_args_list, gain_flags, 
```

Comment:
I don't think it makes sense to deprecate this function in this PR if we aren't deprecating the H1C pipeline just yet. Instead of removing this function, I'd keep it around but place a deprecation warning on it

Reply:
Done.